src/encoded/static/components/facets/exists.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { DefaultExistsFacet } from './defaults';
+import FacetRegistry from './registry';
+
+
+/**
+ * The Publication datasets.accession facet field renders an "exists" facet.
```

Comment:
It seems a little odd to only mention the `datasets.accession` facet field here when this file registers two exists type facets (also the `nih_institutional_certifications` facet).

Reply:
Agreed.